emperor/util.py

```diff
@@ -250,29 +250,30 @@ def keep_columns_from_mapping_file(data, headers, columns, negate=False):
             continue
 
     # keep the elements at the positions indices
-    keep_elements = lambda elements, indices :[element for i, element in
-        enumerate(elements) if i in indices]
+    def keep_elements(elements, indices):
+        return [element for i, element in enumerate(elements) if i in indices]
 
     headers = keep_elements(headers, indices_of_interest)
     data = [keep_elements(row, indices_of_interest) for row in data]
 
     return data, headers
 
+
 def preprocess_coords_file(coords_header, coords_data, coords_eigenvals,
-                        coords_pct, mapping_header, mapping_data,
-                        custom_axes=None, jackknifing_method=None,
-                        is_comparison=False,
-                        pct_variation_below_one=False):
+                           coords_pct, mapping_header, mapping_data,
+                           custom_axes=None, jackknifing_method=None,
+                           is_comparison=False,
+                           pct_variation_below_one=False):
     """Process a PCoA data and handle customizations in the contents
 
     Inputs:
     coords_header: list of sample identifiers in the PCoA file _or_ list of
     lists with sample identifiers for each coordinate file (if jackknifing or
-    comparing plots)
-    coords_data: matrix of coordinates in the PCoA file _or_ list of numpy
-    arrays with coordinates for each file (if jackknifing or comparing plots)
-    coords_eigenvals: numpy array with eigenvalues for the coordinates file _or_
-    list of numpy arrays with the eigenvalues (if jackknifing or comparing plots
+    comparing plots) coords_data: matrix of coordinates in the PCoA file _or_
```

Comment:
Yes, really what we should be doing is following NumPy-doc guidelines
here, but since these functions are not intended to be used outside the
project, that's not high priority at the moment. The only module in the
project that's intended to be used outside the project is core.py.

On (Mar-07-15|11:06), Santiago Torres wrote:
> >      """Process a PCoA data and handle customizations in the contents
> >  
> >      Inputs:
> >      coords_header: list of sample identifiers in the PCoA file _or_ list of
> >      lists with sample identifiers for each coordinate file (if jackknifing or
> > -    comparing plots)
> > -    coords_data: matrix of coordinates in the PCoA file _or_ list of numpy
> > -    arrays with coordinates for each file (if jackknifing or comparing plots)
> > -    coords_eigenvals: numpy array with eigenvalues for the coordinates file _or_
> > -    list of numpy arrays with the eigenvalues (if jackknifing or comparing plots
> > +    comparing plots) coords_data: matrix of coordinates in the PCoA file _or_
> 
> After looking at this, wouldn't some white-space between each "input"/"output" help for readability?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/358/files#r25998530